bridgesync/downloader.go

```diff
@@ -74,6 +74,9 @@ func buildAppender(client EthClienter, bridge common.Address, syncFullClaims boo
 			Amount:             bridge.Amount,
 			Metadata:           bridge.Metadata,
 			DepositCount:       bridge.DepositCount,
+			BlockTimestamp:     b.Timestamp,
+			TxHash:             l.TxHash,
+			FromAddress:        l.Address,
```

Comment:
Rename it per Notion doc (or the `ContractAddress`), wdyt?
```suggestion
			EventInitiatorAddress:        l.Address,
```

Reply:
i would like to go with fromAddress. it can be used by frontend in whatever way they want then. wdyt?